first-demo/my-block.js

```diff
@@ -1,6 +1,6 @@
+import { registerBlockType } from '@wordpress/blocks';
```

Comment:
I can not able to merge your request because i am using the simple js.
{  } is not allowed in the this Demo.
it will used in ES next versions.